database/migrations/2019_08_27_161055_remove_excess_reminder_fields_from_registrations.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class RemoveExcessReminderFieldsFromRegistrations extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        // Cannot bulk drop multiple tables with SQLite (tests).
+        Schema::table('registrations', function($table) {
```

Comment:
Ooh, apparently you can do a multi column drop by putting the names into an array and putting that in one 

https://laravel.com/docs/5.5/migrations#dropping-columns